app/src/main/kotlin/com/wire/android/ui/common/textfield/WirePasswordTextField.kt

```diff
@@ -81,7 +81,7 @@ fun WirePasswordTextField(
         trailingIcon = {
             val image = if (passwordVisibility) Icons.Filled.Visibility else Icons.Filled.VisibilityOff
             IconButton(onClick = { passwordVisibility = !passwordVisibility }) {
-                Icon(imageVector = image, "", modifier = Modifier
+                Icon(imageVector = image, "Show or hide Password", modifier = Modifier
```

Comment:
Can you please extract the contentDescription to String Resource, so it can be localized, and I think it needs to be added to the `WireTextField` not the icon